dashboard/app/views/pd/teacher_application/teacher_application.html.haml

```diff
@@ -0,0 +1,15 @@
+- content_for(:head) do
+  = stylesheet_link_tag 'css/pd', media: 'all'
+
+-# We have to render the top outside react. It would be nice to move the school dropdown to a react component
+%script{src: minifiable_asset_path('js/teacherApplication.js')}
+
+%h2
+  Section 1: Which Program is Right for you?
+
+= render partial: 'shared/school_info', locals: {form_name: "school_info"}
+
+%div#application-container
+
+:javascript
+  window.ReactDOM.render(React.createElement(window.TeacherApplication), document.getElementById('application-container'))
```

Comment:
This is the one line you would put in the `src/sites/studio/pages/pd/teacher_application/teacher_application.js` file.  Actually there would be four, since now you get to be a lot more explicit about what libraries you are depending on:

```jsx
import React from 'react';
import ReactDOM from 'react-dom';
import TeacherApplication from '@cdo/apps/code-studio/pd/teacher_application/TeacherApplication';

ReactDOM.render(<TeacherApplication/>, document.getElementById('application-container'));
```